go.mod

```diff
@@ -3,9 +3,15 @@ module github.com/microsoft/go-mssqldb
 go 1.13
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.19.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.11.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.21.1
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.13.1
```

Comment:
can we go to v0.14.0 per request from #29 ?

Reply:
we could...but there are several additional breaking changes in making that move

Reply:
AAD has to be the most fluid dependency in the history of dependencies

Reply:
well let's make that move separately then